src/material-experimental/mdc-slider/slider.ts

```diff
@@ -529,6 +529,7 @@ const _MatSliderMixinBase:
     '[class.mdc-slider--disabled]': 'disabled',
     '[class.mdc-slider--discrete]': 'discrete',
     '[class.mdc-slider--tick-marks]': 'showTickMarks',
+    '(mouseenter)': '_onMouseEnter()',
```

Comment:
Can we get away with doing this on `mousedown` or is that going to be too late? If not, `mouseenter` should work as well, but we have had issue reports in the past about other components triggering change detections if somebody accidentally hovers over them, but doesn't actually interact.

Reply:
Good call! I was able to change it to `pointerdown`

Reply:
We've been avoiding `pointerdown` in other places, because it isn't supported in all browsers.

Reply:
what if we do it on mousedown in capture phase?

Reply:
For some reason I can't get it to work with `mousedown`, only with `pointerdown`. Would it be ok for me to copy this check that MDC uses for deciding whether or not to use `pointer` events? ([link](https://github.com/material-components/material-components-web/blob/master/packages/mdc-slider/foundation.ts#L47))

Something like this:

```ts
_onPointerDown(): void {
    if (SUPPORTS_POINTER_EVENTS) {
        this._foundation.layout();
    }
}

_onMouseEnter(): void {
    if (!SUPPORTS_POINTER_EVENTS) {
        this._foundation.layout();
    }
}

Reply:
Isn't the `mousedown` going to be redundant if you didn't manage to replace the `pointerdown` with it?

Reply:
I guess the only other choices would be either something like this:

```ts
constructor() {
    const evtType = SUPPORTS_POINTER_EVENTS ? 'pointerdown' : 'mouseenter';
    this._elementRef.nativeElement.addEventListener(evtType, () => {
        this._foundation.layout();
    });
}
```

or to just call `layout` on `mouseenter` like we had it before

Reply:
Ah sorry, I didn't notice that it was `mouseenter`, not `mousedown`. Yes, I think that would work.